ouroboros-network-api/src/Ouroboros/Network/PeerSelection/PeerSharing/Type.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
We can also rename this module to just `Ouroboros.Network.PeerSelection.PeerSharing`